ui/raidboss/data/05-shb/raid/e12n.txt

```diff
@@ -6,5 +6,154 @@ hideall "--sync--"
 
 0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
 
-0 "Start"
+# -ii 4B47 4B1D 5865 585B 586B 5874 5875 5876 587E 587F 5882 5884 5889 588A 588B
+# -ic "Lissom Sculpture" "Beastly Sculpture"
+
+
 0.0 "--sync--" sync /:Engage!/ window 0,1
+1.0 "--sync--" sync /:Eden's Promise:4B1D:/ window 1,1
+14.9 "Maleficium" sync /:Eden's Promise:5872:/ window 14.9,5
+22.1 "--sync--" sync /:Eden's Promise:5879:/
+25.3 "Initialize Recall" sync /:Eden's Promise:5859:/
+34.9 "Judgment Jolt" sync /:Eden's Promise:585F:/
+45.6 "Temporary Current" sync /:Eden's Promise:585C:/
+56.3 "Judgment Jolt" sync /:Eden's Promise:585F:/
+67.0 "Temporary Current" sync /:Eden's Promise:585C:/
```

Comment:
Does this really happen twice in a row, every time??

Reply:
Every log I've seen, yes.